packages/hash/frontend/src/blocks/page/BlockView.tsx

```diff
@@ -216,49 +217,58 @@ export class BlockView implements NodeView<Schema> {
     const blockDraftId = this.getBlockDraftId();
 
     this.renderPortal(
-      <BlockViewContext.Provider value={this}>
-        <CollabPositionIndicators blockEntityId={blockEntityId} />
-        <BlockHighlight blockEntityId={blockEntityId} />
-        <BlockHandle
-          deleteBlock={this.deleteBlock}
-          entityStore={this.store}
-          draftId={blockDraftId}
-          ref={this.blockHandleRef}
-          onMouseDown={() => {
-            /**
-             * We only want to allow dragging from the drag handle
-             * so we set a flag which we can use to indicate
-             * whether a drag was initiated from the drag handle
-             *
-             * @todo we may not need this – we may be able to get
-             *       it from the event
-             */
-            this.allowDragging = true;
-
-            this.dragging = true;
-            this.dom.classList.add(styles["Block--dragging"]!);
-
-            const { tr } = this.editorView.state;
-
-            /**
-             * By triggering a selection of the node, we can ensure
-             * that the whole node is re-ordered when drag & drop
-             * starts
-             */
-            tr.setSelection(
-              NodeSelection.create<Schema>(
-                this.editorView.state.doc,
-                this.getPos(),
-              ),
-            );
-
-            this.editorView.dispatch(tr);
-
-            this.update(this.node);
-          }}
-          onClick={this.onDragEnd}
-        />
-      </BlockViewContext.Provider>,
+      <BlockContext.Consumer>
```

Comment:
hide whitespace to ignore the nesting changes
<img width="688" alt="Screenshot 2022-08-11 at 09 37 58" src="https://user-images.githubusercontent.com/37743469/184095633-f2e90b54-2081-4d9e-8831-a80d08126dfd.png">